assets/js/config.js

```diff
@@ -1,29 +1,46 @@
-module.exports = {
-    api: {
-        article_publication_status: '/api/article_publication_status',
-        queue_article_publication: '/api/queue_article_publication',
-        article_scheduled_status: '/api/article_scheduled_status',
-        article_schedule_for_range: '/api/article_schedule_for_range',
-        schedule_article_publication: '/api/schedule_article_publication',
-        current: '/api/current',
-        article: '/api/article'
-    },
-    // Uncomment to point to test_services.py
-    // api: {
-    //     article_publication_status: 'http://localhost:8008/api/article_publication_status',
-    //     queue_article_publication: 'http://localhost:8008/api/queue_article_publication',
-    //     article_scheduled_status: 'http://localhost:8008/api/article_scheduled_status',
-    //     article_schedule_for_range: 'http://localhost:8008/api/article_schedule_for_range',
-    //     schedule_article_publication: 'http://localhost:8008/api/schedule_article_publication',
-    //     current: 'http://localhost:8008/api/current',
-    //     article: 'http://localhost:8008/api/article'
-    // },
-    logLevel: 0,
-    ISPP: false,
-    colorAdvanceArticle: '#f1f1f1',
-    colorArticle: '#cde1f1',
-    colorText: '#111',
-    errors: {
+module.exports = function config(options) {
+    var _ = require('underscore');
+    var hasOpts = (typeof options != 'undefined') ? true : false;
```

Comment:
If no falsey value is a valid value of options, then this would be simpler as `var hasOpts = !!options;`